integrationtests/tests/fixtures/mod.rs

```diff
@@ -998,13 +1004,26 @@ impl FederationTest {
             server.borrow_mut().fedimint.run_empty_epochs = epochs as u64;
         }
 
-        self.await_consensus_epochs(epochs).await.unwrap();
+        self.run_consensus_epochs_wait(epochs).await.unwrap();
     }
 
-    /// Runs a consensus epoch
+    /// Process n consensus epoch. Wait for events triggering them in case of empty proposals.
+    ///
     /// If proposals are empty you will need to run a concurrent task that triggers a new epoch or
-    /// it will wait forever
-    pub async fn await_consensus_epochs(&self, epochs: usize) -> anyhow::Result<()> {
+    /// it will wait forever.
+    ///
+    /// When conditions triggering proposals are already in place, calling this functions has
+    /// the same effect as calling [`run_consensus_epochs`], as blocking conditions can't
+    /// happen. However in that situation calling [`run_consensus_epochs`] is preferable, as
+    /// it will snappily panic, instead of handing in case of bugs.
```

Comment:
I don't understand the above sentence. Do you mean something along the lines of
```
 However,  in that situation calling [`run_consensus_epochs`] is preferred because it will panic instead of (waiting|handling) in case of bugs.
```
Not sure what you mean by *handing*?